templates/candidates-single.html

```diff
@@ -35,7 +35,7 @@ <h5 class="entity__term__label">District</h5>
             <h5 class="entity__term__label">
               <a class="term" data-term="Principal Campaign Committee">Principal Committee</a>
             </h5>
-            {% for c in committees if c.designation == 'P' %}
+            {% for c in committees_authorized if c.designation == 'P' %}
```

Comment:
To loop over all principal committees, use `for c in committee_groups['P']`. I noticed that we were looping over all committees several times, so I moved that logic into a `groupby` in `views.py`.